task_1/marina.sirotkina/task_1/src/main/java/com/ControlCommands.java

```diff
@@ -0,0 +1,79 @@
+package com;
+
+import java.io.IOException;
+import java.nio.file.StandardOpenOption;
+
+import static com.ControlVersions.getVersionFileURL;
+
+/**
+ * Created by Марина on 12.08.2017.
```

Comment:
Don't leave this comment here. It is redundant, since the information about the author and the date exists in the git commit.